src/Console/InfectionContainer.php

```diff
@@ -221,103 +336,57 @@ public function withDynamicParameters(
         bool $onlyCovered,
         string $formatter,
         bool $noProgress,
-        string $existingCoveragePath,
-        string $initialTestsPhpOptions,
+        ?string $existingCoveragePath,
+        ?string $initialTestsPhpOptions,
         bool $ignoreMsiWithNoMutations,
-        float $minMsi,
-        float $minCoveredMsi
+        ?float $minMsi,
+        ?float $minCoveredMsi,
+        ?string $testFramework,
+        ?string $testFrameworkOptions
```

Comment:
One big change is happening here:

- dynamic parameter is about the schema file so we configure it here
- the other dynamic parameters are now about the "domain" config (`Configuration`) which we can set here
- all the other services depending of those dynamic parameters no longer do: they depend on the domain config which has the desired and ready to consume values. There is no longer a dependency on this external input or trying to sort out which value to take (from the input or the config) - which drastically simplify things here